controller.go

```diff
@@ -1363,3 +1364,27 @@ func (c *controller) IsDiagnosticEnabled() bool {
 	defer c.Unlock()
 	return c.DiagnosticServer.IsDiagnosticEnabled()
 }
+
+func (c *controller) iptableEnabled() bool {
```

Comment:
Minor nit, but this should probably be called `iptablesEnabled()` (note the s) so that this function comes up when a future engineer is likely to `git grep iptables`.